tests/test_core/test_alignment.py

```diff
@@ -244,6 +244,16 @@ def setUp(self):
         self.b = Alignment(["AAA", "AAA"])
         self.c = SequenceCollection(["AAA", "AAA"])
 
+    def test_deepcopy(self):
```

Comment:
In this test, you should also exercise using the `sliced=True` argument. All we care about for this test case is that it does not fail. The real test of when it works is in the other test method below